src/api/app/controllers/webui/package_controller.rb

```diff
@@ -81,6 +81,7 @@ def show
 
     @comments = @package.comments
     @requests = []
+    @services = Service.find(project: @project.name, package: @package.name)
```

Comment:
Oh! Shity mistake... ActiveXML, always pretending to be an ActiveRecord model :smile: 

Thanks for your senior comment @coolo :smiling_imp: